invenio_files_rest/helpers.py

```diff
@@ -144,7 +145,7 @@ def send_stream(
     if as_attachment or mimetype == "application/octet-stream":
         # See https://github.com/pallets/flask/commit/0049922f2e690a6d
         try:
-            filenames = {"filename": filename.encode("latin-1")}
+            filenames = {"filename": filename.encode("latin-1").decode("utf-8")}
```

Comment:
I feel this is fine, since I remember that `werkzeug` started dealing with encoding/decoding these values a bit better in later versions (which is the reason we've copy-pasted some of this logic here).

It might be even better to remove the logic entirely... but this needs some testing probably

Reply:
rethinking the encode/decode part should be done, but not know. this would need some further testing and would delay the process of moving to flask>=3.0.

Reply:
i could use [this](https://github.com/pallets/werkzeug/blob/main/src/werkzeug/utils.py#L456-L465) instead of what we have now

Reply:
would [that](https://github.com/inveniosoftware/invenio-files-rest/pull/315/commits/d0a6fe692dfbcdaf7e69298ee74b32bdc16faa35) what you had in mind?

Reply:
Yes, that would work as well. I'm still a tiny bit curious why we have to do this on our side, instead of just delegating to Flask/werkzeug (but that can be addressed separately)